Makefile

```diff
@@ -0,0 +1,89 @@
+###
+# This Makefile provides portable, stack-agnostic commands (unlike npm)
+#
+# These commands are meant to be run from your local machine, not from
+# within the docker container. Each command handles exec'ing into the
+# docker container to run the command
+
+ifneq (,)
+	This makefile requires GNU Make.
+endif
+# #######
+# Basics
+platform-onboarded:
+	make migrate-docker
+	docker exec -it platform_platform_1 /bin/bash
+	docker exec -it platform_platform_1 npm run dev
+
+platform-exec:
+	@docker exec -it platform_platform_1 bash
+exec:platform-exec
+
+# ######
+# DB
+
+
+seed-docker:
+	docker exec -it platform_platform_1 /bin/bash
+	composer migrate
+
+seed:
+	make seed-docker
+
+
+migrate:
+	make migrate-docker
+
+
+migrate-docker:
+	docker exec -it platform_platform_1 php artisan migrate:refresh -vvv
+
+db-total-reset:
+	@echo "🔥 Resetting DB"
+	docker exec -it platform_platform_1 composer dump-autoload
+	make migrate-docker
+	make seed-docker
+dbtr:db-total-reset
+
+###
+tinker:
+	php artisan tinker
+
+tinker-docker:
+	docker exec -it platform_platform_1 php artisan tinker
+
+# ######
+# CLIs
+redis-cli:
+	docker exec -it platform_redis_1 redis-cli
+
+mysql-cli:
+	docker exec -it platform_mysql_1 /bin/bash
+	mysql -u root -p
+
+# ###########
+# TESTING
+test:
+	vendor/bin/phpunit
+
+test-docker:
+	docker exec -it platform vendor/bin/phpunit
```

Comment:
the full tests are actually run with `composer run test` , but that expects the API to run on port 8000 , which is why this is what is run in the CI : https://github.com/ushahidi/platform/blob/develop/docker/test.run.sh#L12

Reply:
I fixed this too, I was running test differently. but now it works fine
so you run this command  `@docker exec -it platform_platform_1 bash` to get inside the platform container, and then `composer run test` to run tests inside the platform docker container